prprocessor/config/repos.yaml

```
@@ -4,7 +4,17 @@
 #   redmine_required: enforce commit message format to include Redmine ticket number (default: false)
 #   refs: optional list of Redmine projects that commits here can "refs" (default: [])
 #   branches: list of primary development branches, else uses repo default
+#   exempt_patterns: optional list of regex patterns for commits that should bypass
+#                    Redmine validation (useful for i18n, automated updates, etc.)
 ---
+
+# Global default patterns applied to ALL repositories (unless overridden)
+global_config:
+  exempt_patterns:
+    - "^i18n - "
+    - "^Localization - "
+    - "^Update translations"
+    - "^Automatic locale update"
```

I don't like the many variations for this. In our code base we only have 2 and let's keep it that way
```suggestion
    - "^i18n - (extracting new, )?pulling from tx$"
```

Sources:
* https://github.com/theforeman/foreman/blob/ee1afb939ebfea220177b5066c4bb9744721cdfe/locale/Makefile#L62
* https://github.com/theforeman/foreman_plugin_template/blob/f92b585db9ddd4d8fc9c2bc3209162962157a67c/locale/Makefile#L70
* https://github.com/theforeman/hammer-cli/blob/737ec089d22d01aa537764dc9b6be0f1b8718a64/lib/hammer_cli/i18n/find_task.rb#L92

---

Right, sort of wanted to have a discussion about this - if the `i18n` one is the only one we want to have in at this moment. 

I'll leave only that one there then.